client/src/app/login/login.component.html

```diff
@@ -17,7 +17,7 @@
           or create an account
         </a>
 
-        <a i18n *ngIf="signupAllowed === false" href="https://joinpeertube.org/en/#getting-started" target="_blank" class="create-an-account">
+        <a i18n *ngIf="signupAllowed === false" href="https://joinpeertube.org/en/#getting-started" target="_blank" alt="Link to joinpeertube.org Getting started" title="Click here to see how to get started!" class="create-an-account">
```

Comment:
`<a>` tags does not have an `alt` attribute in HTML specification

Reply:
Yep, it's exactly what I was looking for… I don't why I was sure they had! :/ Will fix it.